code/modules/shuttle/emergency.dm

```diff
@@ -631,6 +631,20 @@
 	. = ..()
 	initiate_docking(SSshuttle.getDock("emergency_home"))
 
+/obj/machinery/computer/emergency_shuttle/AltClick(user)
+	if(isliving(user))
+		xeno_hijack_check(user)
+
+/obj/machinery/computer/emergency_shuttle/proc/xeno_hijack_check(mob/living/user)
+	if(isalien(user) && user.mind?.has_antag_datum(/datum/antagonist/xeno/queen) && (user in SSshuttle.hostileEnvironments))
+		priority_announce("Corruption detected in \
+						shuttle controll protocols. Please contact your \
+						supervisor.", 
+	"SYSTEM ERROR", 'sound/voice/alien_queen_roar.ogg')
+		sleep(50)
```

Comment:
use timer

mirroring TG:
this can be spammed if you just keep alt clicking the console

Reply:
Done